empack/toposort.py

```diff
@@ -15,15 +15,18 @@ def toposort(pkg_metas):
         name = pkg_meta['name']
         g.add_node(name)
         name_to_meta[name] = pkg_meta
-    
-    for pkg_meta in pkg_metas:  
+
+    for pkg_meta in pkg_metas:
         name = pkg_meta['name']
         depends = [d.split(' ')[0] for d in pkg_meta['depends']]
 
         for d in depends:
             g.add_edge(d, name)
 
+    try:
+        sorted_nodes = [name_to_meta[n] for n in nx.topological_sort(g)]
+    except nx.exception.NetworkXUnfeasible as e:
+        print(f'Failed to perform the topological sorting on {g.graph}')
```

Comment:
this will just print the dimensions of the graph 
```
>>> print(G)
DiGraph with 2 nodes and 1 edges
```

Reply:
no?
```python
In [1]: import networkx as nx

In [2]: G = nx.DiGraph(day="Friday")

In [3]: G
Out[3]: <networkx.classes.digraph.DiGraph at 0x7f5c035a9c30>

In [4]: G.graph
Out[4]: {'day': 'Friday'}
```